istioctl/cmd/proxyconfig.go

```diff
@@ -488,18 +568,28 @@ func proxyConfig() *cobra.Command {
   istioctl proxy-config endpoint <pod-name[.namespace]> --cluster "outbound|9411||zipkin.istio-system.svc.cluster.local" -o json
   # Retrieve full endpoint with the status (healthy).
   istioctl proxy-config endpoint <pod-name[.namespace]> --status healthy -ojson
+
+  # Retrieve endpoint summary without using Kubernetes API
+  curl <host>:15000/clusters?format=json > envoy-clusters.json
```

Comment:
a bit disappointing this is a different input file, which was previously hidden to users. I don't see anyway around that though

Reply:
this can only be solved by https://github.com/envoyproxy/envoy/issues/3362

Reply:
Agreed.  There is a `// TODO(fisherxu): migrate this to config dump when implemented in Envoy` in the code for when Envoy 3362 lands.